packages/boot/test/bootstrapTests/liquidation-1.test.ts

```diff
@@ -166,19 +168,17 @@ const checkFlow1 = async (
     await priceFeedDrivers[collateralBrandKey].setPrice(9.99);
 
     // check nothing liquidating yet
-    const liveSchedule: ScheduleNotification = readLatest(
-      'published.auction.schedule',
-    );
+    const liveSchedule = readPublished('auction.schedule');
```

Comment:
**note**: It's nice that the code is cleaner.

I do have mixed feelings about hiding the fact that the code is making a static assumption that isn't runtime verified.